src/Session/Attribute/AttributeBagMirror.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+/**
+ * This file is part of Contao.
+ *
+ * Copyright (c) 2005-2015 Leo Feyer
+ *
+ * @license LGPL-3.0+
+ */
+
+namespace Contao\CoreBundle\Session\Attribute;
+use Symfony\Component\HttpFoundation\Session\Attribute\AttributeBag;
+
+/**
+ * Provides an ArrayAccess mirror for a session AttributeBag.
+ * Used for BC for $_SESSION['FE_DATA'] and $_SESSION['BE_DATA'].
+ *
+ * @author Yanick Witschi <https://github.com/toflar>
+ */
+class AttributeBagMirror implements \ArrayAccess
```

Comment:
I think this should be called `Adapter` instead of `Mirror` to follow the design pattern.